docs/getting-started/flashing-nrf52.md

```diff
@@ -39,12 +39,12 @@ With the latest versions of MacOS, the USB Serial driver is built-in. Do _NOT_ d
   <summary>Removing the CH34x (CH340/CH341) USB Drivers</summary>
   <div>
     <div>
-        If you have already downloaded/installed the MacOS WCH-IC CH340 ("CH341SER_MAC") drivers via the `CH34x_Install_V1.5.pkg`, you will have to Uninstall the kernel extension:
+        If you have already downloaded/installed the MacOS WCH-IC CH340 ("CH341SER_MAC") drivers via the CH34x_Install_V1.5.pkg, you will have to Uninstall the kernel extension:
```

Comment:
Removing backticks here because they are rendered as actual backticks instead of making this stylized. (Too many weird details/div/div, I suspect.)